doc/services.rst

```diff
@@ -8,6 +8,19 @@ Commissaire's message bus and provide/consume services.
 Example Use Cases
 -----------------
 
+Commissaire Container Manager
+``````````````````````````````
+Commissaire's ``Container Manager`` service is a set of long running processes which provide
+a consistant API to work with container managers.
+
+
+Example Configuration File
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+By default ``commissaire-clustermgr-service`` looks at ``/etc/commissaire/containermgr.conf``
+for it's configuration. This can be overriden with the ``-c/--config`` flag.
```

Comment:
As an aside, `read_config_file()` in `commissaire` still defaults to `/etc/commissaire/commissaire.conf`, which applies to all services if `-c/--config` is not given.  Is that still okay, or do want to bake these service-specific defaults into the services?  Then we could remove the `-c` from the `.service` files.

The documentation as written here isn't quite true.